test/spec/moddle.js

```diff
@@ -101,7 +101,10 @@ describe('moddle', function() {
 
       expect(descriptor.ns).to.jsonEqual(expectedDescriptorNs);
       expect(descriptor.properties).to.jsonEqual(expectedDescriptorProperties);
-      expect(descriptor.propertiesByName).to.jsonEqual(expectedDescriptorPropertiesByName);
+      expect(descriptor.propertiesByName.id).to
```

Comment:
Why remove the original check here?

I'd say we could add an additional test case, if we want:

```suggestion
      expect(descriptor.propertiesByName.id).to.equal(descriptor.propertiesByName['props:id']);
```

Reply:
> Why remove the original check here?

Because I added a property and comparing the whole structure depends on the order. That's why I created two `expect`s for this.

> I'd say we could add an additional test case, if we want:

:+1: [added](https://github.com/bpmn-io/moddle/pull/39/commits/d6f342b2bddddc89fa8f51d3a101095d0e8aaf4a)

I applied the suggestion and CI action failed with an empty error message. Fixed it [here](https://github.com/bpmn-io/moddle/pull/39/commits/66bb9e0b9e53b63ce33fd61aaa388ccc6119166d).